packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -104,9 +103,29 @@ class DatePickerInput extends React.Component {
   constructor(props) {
     super(props);
 
+    const format = DateUtil.getDateFormat(props.intl.formatMessage({ id: 'Terra.datePicker.dateFormatOrder' }));
+    const tempDate = DateUtil.convertToISO8601(props.value, format);
+
+    this.state = {};
+
+    if (tempDate) {
+      this.state = {
+        day: tempDate.substr(8, 2),
```

Comment:
> [`String.prototype.substr`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr) is considered a legacy function and should be avoided when possible. 

You may want to use [`substring()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring) method instead.

Reply:
Changed! [89c3693](https://github.com/cerner/terra-framework/pull/1004/commits/89c3693f31162adfc53f90f681cdfda21588b6b0)